test/e2e/petset.go

```diff
@@ -889,6 +889,9 @@ func newPetSet(name, ns, governingSvcName string, replicas int32, petMounts []ap
 		ObjectMeta: api.ObjectMeta{
 			Name:      name,
 			Namespace: ns,
+			Annotations: map[string]string {
+				"pod.alpha.kubernetes.io/initialized": "false",
```

Comment:
I think we can let it be for now and think about it when we're updating the examples?